README.md

```diff
@@ -20,46 +20,58 @@ deployed on [IPFS](https://docs.ipfs.io/concepts/case-study-fleek/) and the Flow
 ```
 
-Once you have the project files, you'll find setup instructions in the `README.md` files inside each folder.
+### 3. Configure projects
+
+Follow the setup instructions in the `README.md` files inside each folder to set up each project's environment variables.  
 
-### 4. Deploy the contracts
+### 4. Start the Flow Emulator
 
-Follow the [deployer instructions](https://github.com/onflow/kitty-items/blob/master/kitty-items-deployer) to deploy the Kitty Items Cadence contracts to your Testnet account.
+Start the Flow emulator by running the following command in the project's root directory: 
+```
+flow project start-emulator 
+```
+
+### 5. Deploy contracts
+With the emulator running you can deploy the Kitty Items contracts. Run the following command form the project's root directory:
+```
+flow project deploy-contracts 
+```
 
-### 5. Run the API
+### 5. Start the API
 
 After the contracts are deployed, follow the [kitty-items-js instructions](https://github.com/onflow/kitty-items/tree/master/kitty-items-js)
 to install and run the Kitty Items API. This backend service is responsible for initializing accounts, minting NFTs, and processing events.
 
-### 6. Launch the web app
+### 6. Start the dev-wallet
+
+When developing locally, you'll need an emulated wallet as well. The wallet let's you create accounts and sign transactions as user's other than the emulator's service account.
+
+Run this command to start the dev-wallet from the `fcl-dev-wallet` directory: 
+```
+npm install && npm run start:dev
+```
+
+### 7. Launch the web app
 
 Lastly, follow the [kitty-items-web instructions](https://github.com/onflow/kitty-items/tree/master/kitty-items-web) to launch the Kitty Items front-end React app.
 
 ## Project Overview
 
-![kitty-items-overview](https://user-images.githubusercontent.com/901466/106943245-0eac2b00-66da-11eb-960e-a1db5b1d028d.png)<!-- .element width="100%" -->
+<img width="100%" alt="Screen Shot 2021-02-25 at 6 24 05 PM" src="https://images.zenhubusercontent.com/583c7559b9dc3c6220229d4b/5357e2a4-8ae9-4b3d-9e7a-329f12bbf2ad">
 
 ## 🔎 Legend
 
 Above is a basic diagram of the parts of this project contained in each folder, and how each part interacts with the others.
 
-### 1. The Flow Blockchain
+### 1. The Flow Emulator
 
-Flow is a next-generation blockchain platform that is designed to be fast and easy to use. If you're new to Flow, consider reading more about it in the [Flow Primer](https://www.onflow.org/primer), or visit the [developer documentation](https://docs.onflow.org/).
+Flow is a next-generation blockchain platform that is designed to be fast and easy to use. If you're new to Flow, consider reading more about it in the [Flow Primer](https://www.onflow.org/primer), or visit the [developer documentation](https://docs.onflow.org/). The emulator simulates Flow's operatio in a singel process you can run locally while developing. Read about the emulator [here](https://docs.onflow.org/emulator)
```

Comment:
Typo - should be `The emulator simulates Flow's operation in a single process`

Reply:
Nice catch thank you.

Reply:
Nice catch thank you.